richtextfx-demos/src/main/java/org/fxmisc/richtext/demo/hyperlink/HyperlinkOps.java

```diff
@@ -0,0 +1,41 @@
+package org.fxmisc.richtext.demo.hyperlink;
+
+import org.fxmisc.richtext.model.SegmentOpsBase;
+
+public class HyperlinkOps<S> extends SegmentOpsBase<Hyperlink<S>, S> {
```

Comment:
I added that in #553 since I realized that it's easier just to tell developers to extend a base class that already properly overrides everything than to tell them to copy/paste the same pattern in the wiki page.

No release currently has that, but you can get it if you follow the Jitpack instructions on the ReadMe.